components/layout_2020/table/layout.rs

```diff
@@ -78,7 +78,7 @@ impl CellLayout {
             .layout
             .fragments
             .iter()
-            .any(|fragment| !matches!(fragment, Fragment::AbsoluteOrFixedPositioned(_)))
+            .all(|fragment| matches!(fragment, Fragment::AbsoluteOrFixedPositioned(_)))
```

Comment:
This is only removing one negation, you need to remove the one in `!self` too.
That is, checking that there isn't any of the fragments which is different than AbsoluteOrFixedPositioned is equivalent to checking that all of the fragments are AbsoluteOrFixedPositioned.